sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/alibaba/alibaba.yaml

```diff
@@ -0,0 +1,102 @@
+provider:
+  metadata:
+    name: Alibaba 
+    category: Cloud provider
+    references:
+      - description: OWASP Top 10 2021 Category A7 - Identification and Authentication Failures
+        link: https://owasp.org/Top10/A07_2021-Identification_and_Authentication_Failures
+        type: Standards
+      - description: OWASP Top 10 2017 Category A3 - Sensitive Data Exposure
+        link: https://www.owasp.org/www-project-top-ten/2017/A3_2017-Sensitive_Data_Exposure
+        type: Standards
+      - description: MITRE, CWE-798 - Use of Hard-coded Credentials
+        link: https://www.owasp.org/www-project-top-ten/2017/A3_2017-Sensitive_Data_Exposure
+        type: Standards
+      - description: MITRE, CWE-259 - Use of Hard-coded Password
+        link: https://cwe.mitre.org/data/definitions/259
+        type: Standards
+      - description: SANS Top 25 - Porous Defenses
+        link: https://www.sans.org/top25-software-errors/#cat3
+        type: Standards
+    description: description.adoc
+    message: Make sure this Alibaba secret gets revoked, changed, and removed from the code.
+  detection:
+    pre:
+      include:
+        paths:
+          - ".env"
+          - ".*env*"
+        ext:
+          - ".config"
+        content:
+          - "alibaba"
+      reject:
+        paths: 
+          - "*/tests?/*"
```

Comment:
I think this is supposed to be a regex? It is not a valid regex.

Reply:
This part is now removed